CHANGELOG.md

```diff
@@ -1,13 +1,13 @@
 # [1.1.0](https://github.com/Stacked-Org/generator/compare/v1.0.0...v1.1.0) (2023-04-03)
 
+## 1.2.0
```

Comment:
This still doesn't make sense. We can't have v 1.2.0 under v 1.1.1. It should be the latest version

Reply:
I thought differently, this has been resolved.